neural_seq_qa/infer.py

```diff
@@ -0,0 +1,82 @@
+import os
+import sys
+import argparse
+
+import paddle.v2 as pd
```

Comment:
I prefer to use `import paddle.v2 as paddle` to keep consistent with all the other PaddlePaddle demos in models and book.

Reply:
All `pd` have been changed to `paddle`